sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultKeyStore.java

```diff
@@ -76,13 +70,26 @@
      */
     private KeyVaultClient keyVaultClient;
 
+    /**
+     * Store certificates on portal info
+     */
+    private final KeyVaultCertificates keyVaultCertificates;
```

Comment:
It's better to put `classpathCertificates` and `classpathCertificates` together.